env0/resource_team_project_assignment.go

```diff
@@ -0,0 +1,91 @@
+package env0
+
+import (
+	"context"
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceTeamProjectAssignment() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceTeamProjectAssignmentCreateOrUpdate,
+		ReadContext:   resourceTeamProjectAssignmentRead,
+		UpdateContext: resourceTeamProjectAssignmentCreateOrUpdate,
+		DeleteContext: resourceTeamProjectAssignmentDelete,
+
+		Schema: map[string]*schema.Schema{
+			"team_id": {
+				Type:        schema.TypeString,
+				Description: "id of the team",
+				Required:    true,
+				ForceNew:    true,
```

Comment:
what `ForceNew` do? 
do we need it?

Reply:
>ForceNew indicates that any change in this field requires the resource to be destroyed and recreated.

https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#forcenew

Reply:
in other words, we want to destroy the existing `teamProjectAssignment` when the teamId was changed.